nixos/modules/services/networking/minidlna.nix

```diff
@@ -108,6 +108,31 @@ in
           Many people prefer shorter announce intervals (e.g. 60 seconds)
           on their home networks, especially when DLNA clients are
           started on demand.
+
+          Instead of waiting on announces, one can open the port UDP 1900
+          to use SSDP discovery. Furthermore setting announce interval
+          to 90000 may be a solution if you are experiencing disconnects
+          with certain clients.
+        '';
+    };
+
+    services.minidlna.inotify = mkOption {
```

Comment:
Please add a `settings` option instead. See https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md for motivation and details. If you need a hand please don't hesitate to say so

Reply:
Can you merge this for the time being? I am planning to create another PR to address structural settings.

Reply:
Once we add the options we shouldn't go and remove them immediately. Are you saying you want these options even if we add a `settings` option as well?

Reply:
Fair point.

Reply:
I've converted entire module to structural settings. However this one utilizes a non-standard `ini` file. Is it possible to generate with `formats.ini`, but without yielding the header part?

Reply:
You're looking for [`lib.formats.keyValue`](https://github.com/NixOS/nixpkgs/pull/150771). Let's get that merged and then you'll have exactly what you need.

Also, can you please look into `mkRenamedOptionModule` for the options you have removed?

This looks **great**, btw. Thanks!

Reply:
Can you also look into PR #155093 ?